avocado/plugins/spawners/podman.py

```diff
@@ -141,25 +212,10 @@ async def spawn_task(self, runtime_task):
         container_id = stdout.decode().strip()
 
         runtime_task.spawner_handle = container_id
-
-        # Currently limited to avocado-runner, we'll expand on that
-        # when the runner requirements system is in place
-        this_path = os.path.abspath(__file__)
-        base_path = os.path.dirname(os.path.dirname(os.path.dirname(this_path)))
-        avocado_runner_path = os.path.join(base_path, 'core', 'nrunner.py')
-        try:
-            # pylint: disable=E1133
-            proc = await asyncio.create_subprocess_exec(
-                podman_bin,
-                "cp",
-                avocado_runner_path,
-                "%s:%s" % (container_id, entry_point_cmd))
-        except (FileNotFoundError, PermissionError):
-            return False
-
-        await proc.wait()
-        if proc.returncode != 0:
-            return False
+        await self._copy_to_container(podman_bin, container_id,
+                                      avocado_egg_src, avocado_egg_dst)
+        await self._copy_to_container(podman_bin, container_id,
+                                      setuptools_egg_src, setuptools_egg_dst)
```

Comment:
Indeed, all of this and they it is run will simplify the deployment of python applications significantly.